package.json

```diff
@@ -29,7 +29,8 @@
     "test": "jake test",
     "build": "jake build",
     "release": "./build/publish.sh",
-    "rollup": "rollup src/Leaflet.js -f umd -n L > bundle.js"
+    "lint": "node_modules/.bin/eslint --fix src/",
```

Comment:
`node_modules/.bin` is not necessary in `package.json` scripts — it's in path automatically.